plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/rest/TrinoRestCatalog.java

```diff
@@ -183,16 +183,7 @@ public void renameNamespace(ConnectorSession session, String source, String targ
     public List<SchemaTableName> listTables(ConnectorSession session, Optional<String> namespace)
     {
         SessionContext sessionContext = convert(session);
-        List<Namespace> namespaces;
```

Comment:
@amogh-jahagirdar pls take a look at the guideline for the commit messages in Trino

https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#format-git-commit-messages

Potential error message (i am aiming for conciseness in the commit title):

`Return empty table listing when non-existent namespace is provided`

Reply:
I updated to `Return empty table list in Iceberg REST Catalog when non-existent namespace is provided`

Reply:
Keep the commit title under 80 chars long.